weave-js/src/components/Form/TextField.tsx

```diff
@@ -78,34 +79,37 @@ export const TextField = ({
     <Tailwind style={{width: '100%'}}>
       <div
         className={classNames(
-          'relative',
-          textFieldSize === 'medium' ? 'h-32' : 'h-40'
+          'relative rounded-sm',
+          textFieldSize === 'medium' ? 'h-32' : 'h-40',
+          'outline outline-1 outline-moon-250',
+          {
+            'hover:outline-2 [&:hover:not(:focus-within)]:outline-[#83E4EB]':
+              !errorState,
+            'focus-within:outline-2 focus-within:outline-teal-400': !errorState,
+            'outline-2 outline-red-450': errorState,
+            'pointer-events-none opacity-50': disabled,
+          }
```

Comment:
these outline/opacity styles were originally applied on [the child div](https://github.com/wandb/weave/pull/2056/files#diff-cfe478a74185bdf97c39dd00bfa08d181cbf7bb6e9d24630ebad18b8f1b17bacL84), but had to move it up because that div doesn't wrap the icon. this ensures correct disabled & hover states are applied to the icon.